.github/workflows/codespaces.yml

```diff
@@ -1,6 +1,9 @@
 name: Java Codespaces Image Build
 
 on:
+  schedule:
+    # Run weekly at 3:00 AM
+    - cron: "0 3 * * 0"
   push:
     branches:
       - master
```

Comment:
🤔 💭 should this be `main` now, or does this still work?

Reply:
d'oh, good point i keep forgetting this.